### PR TITLE
♻️ Redefine custom-elements-v1 as an RTV experiment

### PR DESCRIPTION
**Pull request by jridgewell**
This sidesteps https://github.com/ampproject/amphtml/issues/24499 by using an RTV experiment instead of a local experiment.

Also, it'll show the performance benefits, since we'll be able to DCE the old polyfill code. 😃

### CODE REVIEW

File: build-system/global-configs/experiments-config.json [hunk 1]
```diff
@@ -15,5 +15,12 @@
     "expirationDateUTC": "2019-10-10",
     "defineExperimentConstant": "_RTVEXP_INABOX_LITE"
   },
-  "experimentC": {}
+  "experimentC": {
```
Comment (kristoferbaxter) [inline build-system/global-configs/experiments-config.json]: Should we revisit this keyed name approach in a future PR? There doesn't appear to be any value to this.

File: src/friendly-iframe-embed.js [hunk 2]
```diff
@@ -857,7 +857,11 @@ export class FriendlyIframeEmbed {
 function installPolyfillsInChildWindow(parentWin, childWin) {
   installDocContains(childWin);
   installDOMTokenList(childWin);
-  if (isExperimentOn(parentWin, 'custom-elements-v1') || getMode().test) {
+  if (
+    // eslint-disable-next-line no-undef
```
Comment (kristoferbaxter) [inline src/friendly-iframe-embed.js]: Do you think we should continue to disable via ESLint these lines or should we fix the issue so ESLint can do the right thing by default here. (Example always use a value from a known Object)

Comment (jridgewell) [reply src/friendly-iframe-embed.js]: I can't remember how to declare a global in eslint anymore…

File: build-system/global-configs/experiments-config.json (hunk 1 above)
Comment (jridgewell) [reply build-system/global-configs/experiments-config.json]: These are tied to SCS's serving names, which only has A/B/C reserved.

File: build-system/global-configs/experiments-config.json [hunk 3]
```diff
@@ -15,5 +15,12 @@
...
+    "name": "CEv1",
```
Comment (choumx) [inline build-system/global-configs/experiments-config.json]: Nit: A more readable name please.

File: build-system/global-configs/experiments-config.json [hunk 4]
```diff
@@ -15,5 +15,12 @@
...
+    "environment": "AMP",
+    "command": "gulp dist --defineExperimentConstant=CUSTOM_ELEMENTS_V1",
+    "issue": "https://github.com/ampproject/amphtml/issues/17243",
+    "expirationDateUTC": "2020-12-31",
```
Comment (choumx) [inline build-system/global-configs/experiments-config.json]: I hope we don't take another year to ship. How about +3 months?

File: build-system/global-configs/experiments-config.json (hunk 3 above)
Comment (jridgewell) [reply build-system/global-configs/experiments-config.json]: Done.

File: build-system/global-configs/experiments-config.json (hunk 4 above)
Comment (jridgewell) [reply build-system/global-configs/experiments-config.json]: Lol. I was more annoyed by the recent build breaks. I don't know why we have this defined here.

**Pull request merged**